src/test/java/org/myrobotlab/service/WorkETest.java

```diff
@@ -8,27 +8,25 @@
 import org.junit.AfterClass;
```

Comment:
I don't see any asserts on this test. please put the @Ignore back unless it actually tests something.

Reply:
Heh, good point ;)
I'll test-driven dev .. soon as I figure out what its supposed to do ...